IoTHubManager/Constants.cs

```diff
@@ -15,7 +15,7 @@ public class Urls
             public const string DEPLOYMENTS_PATH = "/deployments/";
 
             // Device Simulation
-            public const string SIMULATION_PATH = "/simulations/1";
+            public const string SIMULATION_PATH = "/simulations";
```

Comment:
Traced usages of this and it seems like you're correct. I wonder how this was missed before though?

Reply:
It started failing in mid Jan, i think